dev-libs/libbpf/libbpf-9999.ebuild

```diff
@@ -27,17 +27,15 @@ DEPEND="${COMMON_DEPEND}
 	sys-kernel/linux-headers"
 RDEPEND="${COMMON_DEPEND}"
 
-PATCHES=(
-	"${FILESDIR}/libbpf-9999-paths.patch"
-)
+PATCHES=( "${FILESDIR}/libbpf-9999-paths.patch" )
```

Comment:
I prefer to keep this as separate lines since it makes diffs cleaner when adding more patches here.

Reply:
Makes sense, I changed it